processor/blockprocessor/block_processor.go

```diff
@@ -2,89 +2,203 @@ package blockprocessor
 
 import (
 	"fmt"
+	"path"
+	"path/filepath"
 
+	"github.com/algorand/go-algorand/config"
+	algodConfig "github.com/algorand/go-algorand/config"
+	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/data/bookkeeping"
+	"github.com/algorand/go-algorand/data/transactions"
 	"github.com/algorand/go-algorand/ledger"
 	"github.com/algorand/go-algorand/ledger/ledgercore"
+	"github.com/algorand/go-algorand/logging"
 	"github.com/algorand/go-algorand/rpcs"
+	"github.com/algorand/indexer/accounting"
 	"github.com/algorand/indexer/processor"
+	iledger "github.com/algorand/indexer/processor/eval"
```

Comment:
nit: consider renaming `iledger` &rarr; `indexerLedger` or something a bit more verbose. Through my own misjudgment, I kept on thinking that `iledger.xyz` are interfaces of some sort.

Reply:
I'll include these changes in another related pr.